recipes/esridump/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "esridump" %}
+{% set version = "1.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/esridump-{{ version }}.tar.gz
+  sha256: 3ea0b856da1a55b327375fb1eec24d08ec2a5eeda8700a9e696159252876dca7
+
+build:
+  skip: true  # [win]
```

Comment:
It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.

Reply:
Thanks @ocefpaf, will submit a fix shortly